notebooks/README.md

```diff
@@ -1,9 +1,11 @@
 # README
 
-1. Make sure that you have Octave installed. 
+1. Make sure that you have Octave installed.
 2. If you have Conda/Jupyter/pip installed, go to step 4.
+Check is Conda installed properly by `conda list` typing into your terminal
 3. Download the [Anaconda Installer](https://www.anaconda.com/products/individual) and install it.
-4. Install [Octave kernel](https://pypi.org/project/octave-kernel/): 
+If using miniconda, run `conda install jupyter` to download and install the Jupyter Notebook package.
+4. Install [Octave kernel](https://pypi.org/project/octave-kernel/):
```

Comment:
I will try all these steps in the brand 'new' old mac to see if it works and if there are faster ways like installing stuff e.g. via homebrew.

Related to this, I fear that these installing steps can be done in different (and faster) ways depending on the OS. What do you think?

Worth to point to different ways depending on the OS?

Reply:
I think anaconda/conda provide very good support on the issue. We should not spend time on it thaaaat much. 
I've tried these with my mac, thus I added the tiny suggestions.

Reply:
Best way is to point to other doc where they have that nailed already but I agree that for OS specific info is nice.

I think I had taken that from Agah's repo from the OHBM hackathon.

Maybe we have some more install info on the hackathon website.

Reply:
But in general I would have an approach of "some doc is better than none and update it when someone points out it is not clear".